components/Banner/src/MDCBannerView.h

```diff
@@ -0,0 +1,48 @@
+// Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <UIKit/UIKit.h>
+
+#import "MaterialButtons.h"
+
+__attribute__((objc_subclassing_restricted)) @interface MDCBannerView : UIView
```

Comment:
Please add class documentation. This should typically include:

- What the class is meant to be used for.
- A link to the Material spec for this component.
- Any meta usage considerations that may not be captured in a single API doc.

Reply:
Addressed in cc88c99719fe310f14bac8e7d4f1b590015c8218.